pkg/registry/common/refresh/nse_registry_client_test.go

```diff
@@ -70,12 +70,30 @@ func (c *checkExpirationTimeClient) Unregister(ctx context.Context, in *registry
 	return new(empty.Empty), nil
 }
 
+func TestIsStable_TestNewNetworkServiceEndpointRegistryClient(t *testing.T) {
```

Comment:
Result for these tests https://github.com/networkservicemesh/sdk/pull/438/commits/d01f2205799b90393daa15158b9a42766d1a4ed4